server/main.lua

```diff
@@ -104,20 +104,21 @@ lib.callback.register('qbx_vehicleshop:server:GetVehiclesByName', function(sourc
     local player = exports.qbx_core:GetPlayer(src)
     if not player then return end
 
-    local vehicles = exports.qbx_vehicles:GetPlayerVehicles({
-        citizenid = player.PlayerData.citizenid
-    })
-
+    local qbx_vehicles = exports.qbx_vehicles
```

Comment:
theres no need to do this. just use the exports normally

Reply:
it just better practice to not access `exports.` every time in a loop. Lua get local var faster